src/Templates/src/templates/maui-blazor/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "Windows Machine": {
       "commandName": "MsixPackage",
-      "nativeDebugging": false
+      "nativeDebugging": true
```

Comment:
This change seems unrelated to adding the new template. I'm not sure what it is, but is it something that we'd want to consider independently of this change?